src/main/java/catan/controllers/PlayController.java

```diff
@@ -101,6 +101,22 @@ public BoughtCardDetails buyCard(@RequestParam(value = "token", required = false
         return new BoughtCardDetails(developmentCardCode);
     }
 
+    @RequestMapping(value = "use-card/year-of-plenty",
+            method = RequestMethod.POST,
+            produces = MediaType.APPLICATION_JSON_VALUE)
+    public void useCardYearOfPlenty(@RequestParam(value = "token", required = false) String token,
+                          @RequestParam("gameId") String gameId,
```

Comment:
if you have splitted params on separate lines, align them accordingly.
use ctrl+L before commit, it will reformat code